app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/item/MessageContentAndStatus.kt

```diff
@@ -61,11 +62,20 @@ internal fun UIMessage.Regular.MessageContentAndStatus(
             onAssetClicked(header.messageId)
         })
     }
+
     val onImageClickable = remember(message) {
         Clickable(enabled = isAvailable, onClick = {
-            onImageClicked(message, source == MessageSource.Self)
+            appLogger.d("MP_CLICK: messageId=${message.header.messageId}")
```

Comment:
It looks like some temporary log, so maybe remove it or give it some more meaningful text 😄